CommonAssemblyInfo.cs

```diff
@@ -15,8 +15,8 @@
 [assembly: AssemblyProduct("Virto Commerce Datatrans Module")]
 [assembly: AssemblyCopyright("Copyright © VirtoCommerce 2017")]
 
-[assembly: AssemblyFileVersion("1.0.6.0")]
-[assembly: AssemblyVersion("1.0.6.0")]
+[assembly: AssemblyFileVersion("1.0.7.0")]
```

Comment:
You don't need to change module version, it will do a release manager when released